filters/openpolicyagent/openpolicyagent.go

```diff
@@ -289,6 +326,37 @@ func (registry *OpenPolicyAgentRegistry) startCleanerDaemon() {
 	}
 }
 
+func (registry *OpenPolicyAgentRegistry) startPluginTriggerDaemon() {
+	interval := registry.pluginTriggerInterval
+
+	// Prevent pods from triggering plugins (f.ex. downloading new bundles) at the same time
+	if registry.maxPluginTriggerJitter > 0 {
+		jitter := time.Duration(rand.Int63n(registry.maxPluginTriggerJitter.Milliseconds())) * time.Millisecond
+		interval += jitter
```

Comment:
I think this jitter might have some unexpected statistical properties: Over time I would expect the interval be independent of the jitter, i.e. after 10 repetitions at aroung 600s +/- jitter. However the jitter is also taken once upon start and then amplifies. Typical jitter implementations would also center around the time point that it would hit without (i.e. use (period - max_jitter, period + max_jitter) as range to pull from. Like here: https://github.com/mroth/jitter

Reply:
I agree with the assessment but IMO this is more about expectation management. I would not expect operational impact if the update/trigger cycles diverge.
Having a more correct implementation will come with either a more complex implementation or the addition of a library.

Reply:
addressed as suggested by resetting the trigger